platform/android/export/export.cpp

```diff
@@ -815,6 +817,11 @@ class EditorExportPlatformAndroid : public EditorExportPlatform {
 							}
 						}
 
+						if (tname == "uses-feature" && attrname == "glEsVersion") {
+
+							encode_uint32(min_gles3 ? 0x00030000 : 0x00020000, &p_manifest.write[iofs + 16]);
```

Comment:
Please review @hpvb, I'm not familiar at all with this manifest binary-hacking code :D